### ISSUE TITLE
Table layout is broken in 0.7.3

### ISSUE THREAD

**Issue by ShyykoSerhiy** [OfficeDev/msteams-ui-components#181]
Table is completely unusable with new flex approach.
Steps to reproduce: 
1. change https://github.com/OfficeDev/msteams-ui-components/blob/master/gh-pages/package.json#L18 to ^0.7.3.
2. yarn
3. yarn dev 
<img width="754" alt="screen shot 2018-08-08 at 12 21 59 am" src="https://user-images.githubusercontent.com/1106995/43816141-46ba7b9e-9aa1-11e8-941c-999230466145.png">

**Comment by MattSFT** [comment]
Agreed. The table implementation is not really that good. Do you have ideas on how to fix it?

**Comment by ShyykoSerhiy** [comment]
We probably can use react virtualized to render tables.

**Comment by MattSFT** [comment]
I'm not too familiar with those, could you send me a link or maybe a PR?
